sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/TextSentiment.java

```diff
@@ -3,43 +3,27 @@
 
 package com.azure.ai.textanalytics.models;
 
-import com.azure.core.annotation.Immutable;
-import com.azure.core.util.ExpandableStringEnum;
-import com.fasterxml.jackson.annotation.JsonCreator;
-
 /**
- * Defines values for {@link TextSentiment}.
+ * Defines enum values for {@link TextSentiment}.
  */
-@Immutable
-public final class TextSentiment extends ExpandableStringEnum<TextSentiment> {
-    /**
-     * Static value Positive for {@link TextSentiment}.
-     */
-    public static final TextSentiment POSITIVE = fromString("positive");
-
+public enum TextSentiment {
```

Comment:
This is a breaking change and shouldn't be done. If this change is only to be consistent with .NET, unfortunately, we can't do it after GA.

Reply:
cc: @JonathanGiles